packages/calcite-components/src/components/tab/tab.e2e.ts

```diff
@@ -20,4 +36,32 @@ describe("calcite-tab", () => {
       { propertyName: "scale", defaultValue: "m" },
     ]);
   });
+
+  it("should allow the CSS custom property to be overridden when applied to :root", async () => {
+    const overrideStyle = "0px";
+    const page = await newE2EPage();
+    await page.setContent(html`
+      <style>
+        :root {
+          --calcite-block-padding: ${overrideStyle};
+        }
+      </style>
+      ${tabBlockPadding}
```

Comment:
This test is misleading because the CSS prop is being set both at the root and component-levels (because of the reused test HTML). Sidebar: [block tests](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/components/block/block.e2e.ts#L289-L292) share the same issue. cc @macandcheese

Reply:
Could you open an issue for that - we can update Block to match the tests here.

Reply:
I'll open it right now.

Reply:
https://github.com/Esri/calcite-design-system/issues/8630